aptos-move/framework/aptos-framework/sources/stake.move

```diff
@@ -429,29 +319,63 @@ module aptos_framework::stake {
         move_to(account, AptosCoinCapabilities { mint_cap })
     }
 
+    /// Initialize the validator account and give ownership to the signing account
+    /// except it leaves the ValidatorConfig to be set by another entity.
+    /// Note: this triggers setting the operator and owner, set it to the account's address
+    /// to set later.
+    public entry fun initialize_owner_only(account: &signer, operator: address, voter: address) acquires OwnerCapability, StakePool {
+        initialize_owner(account, false);
+
+        move_to(account, ValidatorConfig {
```

Comment:
Can we skip creating ValidatorConfig here? Operator needs to call rotate_consensus_key anyway which can create it if needed.

Reply:
No, we cannot, as the operator cannot perform move_to